include/ccl_core.h

```diff
@@ -141,19 +143,28 @@ typedef struct ccl_cosmology
   // other flags?
 } ccl_cosmology;
 
+// Label for whether you are passing a pointer to a sum of neutrino masses or a pointer to a list of 3 masses.
+typedef enum ccl_mnu_convention {
+  ccl_mnu_list=0,   // you pass a list of three neutrino masses
+  ccl_mnu_sum =1,  // sum, defaults to splitting with normal hierarchy
```

Comment:
I consciously close to do this because I think we want a clear default option in C which corresponds to the default option in python. In python, if the user does not indicate how mnu should be split up but passes a single value (rather than a list or array), the sum is split into masses given the normal hierarchy (modulo the issue you pointed out above). But in C we don't have the capability to check the length of mnu, so the user must pass something for the convention. Therefore I wanted to use names that indicate which convention they should pick to get the same as the default behaviour in python. What do you think about this logic?